### ISSUE TITLE
localStorage session expires too early

### ISSUE THREAD

**Issue by shamblesides** [smallbets/userbase#212]
To reproduce:

1. Log in
2. Check localStorage; userbaseCurrentSession will have a correct expirationDate
3. Refresh the page
4. Check localStorage; userbaseCurrentSession will now have an expirationDate that is too early
5. Close and re-open the browser
6. Not logged in!

**Comment by j-berman** [comment]
Are you passing a `sessionLength` param to init as well?

**Comment by shamblesides** [comment]
Oh, I remember reading about how that was an argument in the docs but I completely forgot it! Thanks!

That said, I think it's pretty surprising that not specifying this in the init call will actually _reduce_ the session length when the user logs in.

**Comment by j-berman** [comment]
Agreed. Most sensible default behavior if session length is not specified in init would maintain the session length already set (neither reduce nor extend the session length).

Will keep this open until implemented
